arch/xtensa/src/esp32/esp32_irq.c

```diff
@@ -584,8 +595,8 @@ void up_enable_irq(int irq)
 
   if (irq < XTENSA_NIRQ_INTERNAL)
     {
-      /* Enable the CPU interrupt now for internal CPU. */
 
+      /* Enable the CPU interrupt now for internal CPU. */
```

Comment:
Please revert this small part (it just removes a new line, also if you rebase I think this will conflict since my PR is now merged.)